README.md

```diff
@@ -152,8 +152,15 @@ This code can automatically convert an OpenSim model to the external function us
 	- minimal order of polynomial function. Default is *3* [double]
 - **S.misc.poly_order.upper**: 
 	- maximal order of polynomial function. Default is *9* [double]
+- **S.misc.default_msk_geom_bound**:
```

Comment:
I rather have it named default, since that makes it clear that the other setting takes priority.
I've added more description in readme on how the file should be.